src/components/Menu/Menu.tsx

```diff
@@ -1,36 +1,68 @@
-import React from "react";
+import React, { useContext, useState } from "react";
+import { v4 as uuidv4 } from "uuid";
 import Logo from "../Logo/Logo";
 import SearchBar from "../SearchBar/SearchBar";
 import Button from "../../ui/Button/Button";
 
 import styles from "./Menu.module.scss";
 import ICONS from "../../core/constants/ICONS";
-import I18Y from "../../core/i18y";
 import LanguageToggle from "../LanguageToggle/LanguageToggle";
+import LanguageContext from "../../core/contexts/i18y";
+import MovieForm from "../MovieForm/MovieForm";
+import { MovieProps, putMovie } from "../../core/api";
+import ModalSuccess from "../../ui/ModalSuccess/ModalSuccess";
 
 const Menu = (): JSX.Element => {
-  const onClickHandler = (): void => {
-    // eslint-disable-next-line no-console
-    console.warn("Not (yet) implemented");
+  const { dict } = useContext(LanguageContext);
+  const [showEditor, setShowEditor] = useState<boolean>(false);
+  const [showSuccessModal, setShowSuccessModal] = useState<boolean>(false);
+
+  const submitMovieHandler = async (movie: MovieProps): Promise<void> => {
+    await putMovie(movie);
+    setShowEditor(false);
+    return setShowSuccessModal(true);
   };
 
   return (
-    <div className={styles.menu}>
-      <Logo />
-      <div className={styles.buttons}>
-        <LanguageToggle />
-
-        <Button
-          className={styles.button}
-          text={I18Y().ADD_MOVIE_BTN}
-          icon={ICONS.PLUS}
-          onClick={onClickHandler}
+    <>
+      {showEditor && (
+        <MovieForm
+          id={uuidv4()}
+          title=""
+          url=""
+          genres={[]}
+          overview=""
+          date=""
+          rating=""
+          runtime=""
+          closeFormHandler={() => setShowEditor(false)}
+          formTitle={dict.FORM_HEADER_ADD}
+          onSubmitHandler={submitMovieHandler}
         />
+      )}
+      {showSuccessModal && (
+        <ModalSuccess
+          title={dict.FORM_SUCCESS_TITLE}
+          body={dict.FORM_SUCCESS_BODY_ADD}
+          onDismiss={() => setShowSuccessModal(false)}
```

Comment:
Same as above - it makes sense to have the same way of providing handler functions for JSX as just reference without too much JS code in place.